### ISSUE TITLE
Build version gives error, src class is fine

### ISSUE THREAD

**Issue by mauron85** [xuopled/react-google-maps-loader#1]
Hi, my idea is to build hight order geosuggest component, that can be used as decorator for any other component that accept user input (and has onChange method) like input...

My problem is, when I do import module from npm, I'll end up with error when running and my SearchAddress component is not rendered.

```
warning.js?8a56:45 Warning: o(...): React component classes must extend React.Component.
Uncaught Invariant Violation: o.render(): A valid ReactComponent must be returned. You may have returned undefined, an array or some other invalid object.
```

But when I use raw source is working just fine.

Geosuggest decorator - hight order component (decorated itself by react-google-maps-loader) and accepts any "Searchable" component like input, textarea...

``` javascript
import debounce from 'lodash/debounce';
import React, { Component, PropTypes } from 'react';
import googleMapsLoader from 'react-google-maps-loader';

const geosuggest = (Searchable) => {
  class Geosuggest extends Component {
    constructor(props) {
      super(props);
      this.search = debounce(this._search, props.wait);
      this.state = {
        searchStr: ''
      };
    }

    componentDidMount() {
      const { googleMaps } = this.props;
      console.log(googleMaps);
    }

    handleChange(searchStr = '') {
      this.setState({ searchStr });
      if (searchStr.length >= this.props.minChars) {
        this.search(searchStr);
      }
    }

    _search(searchStr) {
      // TODO: do search and return results
      this.props.onResult(searchStr);
    }

    render() {
      return (
        <div>
          <Searchable
            value={this.state.searchStr}
            onChange={this.handleChange.bind(this)}
          />
        </div>
      );
    }
  }

  Geosuggest.defaultProps = {
    wait: 200,
    minChars: 3,
  };

  Geosuggest.propTypes = {
    wait: PropTypes.number,
    minChars: PropTypes.number,
    googleMaps: PropTypes.object,
    onResult: PropTypes.func.isRequired
  };

  return googleMapsLoader({ libraries: ['places'] })(Geosuggest);
};

export default geosuggest;
```

This is class that is decorated by geosuggest component (adding geosuggest capability)

``` javascript
class SearchAddress extends Component {
  handleResult(result) {
    console.log(result);
  }

  render() {
    const { searchString, handleSearch, onFocus, onBlur, intl } = this.props;
    const Geosuggest = geosuggest(SearchBar);
    return (
      <div>
        <Geosuggest
          wait={500}
          minChars={3}
          onResult={this.handleResult.bind(this)}
        />
      </div>
    );
  }
}
```

**Comment by xuopled** [comment]
Hi @mauron85 , sorry for the delay. I just published a `2.0.0` version. Could you check if that solve your problem ?

**Comment by xuopled** [comment]
I can't reproduce.
